inductiva/tasks/task.py

```diff
@@ -191,6 +191,26 @@ def _format_data_metric(
     def __repr__(self) -> str:
         return str(self)
 
+    def get_task_time(self) -> str:
```

Comment:
This method is not being used and the calculation is being done again in `wait()`.